client/assets/less/forms.less

```diff
@@ -45,35 +42,37 @@
   text-align: center;
   margin-top: 60px;
 }
-input {
-  margin-top: -5px;
-  margin-bottom: 0;
-}
-input[type=text]:focus {
-  border: 3px solid green;
-}
-input[type=text] {
-  border: 1px solid green;
-}
-input[type=email]:focus {
-  border: 3px solid green;
-}
-input[type=email] {
-  border: 1px solid green;
-}
-input[type=password]:focus {
-  border: 3px solid green;
-}
-input[type=password] {
-  border: 1px solid green;
-}
-hr {
-  border-color: @brand-dark;
-  margin: 5% -5%;
-}
-label {
-  color: @brand-dark;
-  font-size: 14px;
+.registration{
```

Comment:
You should use here only 2 selectors, because all of existing are duplicates:
1. input { border: 1px solid green; }
2. input:focus { border: 3px solid green; }
And remove others selectors like input[type=...